styles.css

```diff
@@ -0,0 +1,120 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto+Serif:opsz,wght@8..144,100&display=swap');
+
+@media screen and (max-width: 640px) {
+  main {
+    padding: 0;
+    margin: 0;
+  }
+
+  header,
+  section {
+    margin: 0;
+    padding: 20px;
+    color: #fff;
+  }
+
+  header {
+    background: #3e3c3c;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  #humberger {
+    display: none;
+  }
+
+  header a {
+    font-family: 'Roboto Serif', serif;
+    text-decoration: none;
+    font-weight: bold;
+    font-size: large;
+    color: #fff;
+    padding-left: 2%;
+  }
+
+  #navig {
+    height: fit-content;
+  }
+
+  #hum {
+    font-family: 'Roboto Serif', serif;
+    color: #fff;
+    font-size: 3vh;
+    background-color: #3e3c3c;
+    border: none;
+  }
+
+  #navitems {
+    display: none;
+  }
+
+  #navitems a {
+    box-sizing: border-box;
+    display: block;
+    width: 100%;
+    text-decoration: none;
+    padding-top: 5px;
+    color: #fff;
+    font-weight: 200;
+  }
+
+  #navitems a:hover {
+    background-color: aqua;
+    text-size-adjust: 10%;
+  }
+
+  #toolbar {
+    background-color: #1c1a19;
+    padding-left: 5vh;
+    background-image: url("assets/mobilegroup.svg");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: bottom right;
+    background-position-x: 98%;
+    background-position-y: 70%;
+  }
+
+  #heytext {
+    font-family: 'Crete Round', serif;
+    color: #ff6b00;
+  }
+
+  #whoami {
+    font-family: 'Roboto Serif', serif;
+    color: #fff5e1;
+  }
```

Comment:
- [ ] As requested by the previous reviewer, Kindly make sure to stick to the provided Figma design.  Kindly make sure your design is a close replica of the design provided in the Figma template. This includes font, font size, font family, color, font weight, margin, and padding. :wink: 

  - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/186706876-c8331320-c9a5-4ed7-8c0f-a440688a4860.png)


  - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186706999-3c5ff676-aeff-47de-8366-e5365f868b98.png)